assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -275,6 +321,12 @@ class PageEdit extends Component {
 					</PanelColorSettings>
 					<PanelBody title={ __( 'Background Media', 'amp' ) }>
 						<>
+							{
+								isExcessiveVideoSize &&
+								<Notice status="error" isDismissible={ false } >
+									{ __( 'The video size is more than 1 MB per second.', 'amp' ) }
```

Comment:
Perhaps the `1` here should be pulled from the same constant as `megabyteInBytes` above?

Reply:
Ah, good idea. This is done in https://github.com/ampproject/amp-wp/pull/2509/commits/972e6afba90fd324a908fc02b19d65a261e264bd.

Is this what you had in mind? It's the display with that commit:

<img width="392" alt="size-is-more-than" src="https://user-images.githubusercontent.com/4063887/59082486-ef337e00-88b8-11e9-9d0f-da9eea90d394.png">

Reply:
No, not that it shoes the number of bytes, but just that the underlying megabyte count is stored in a shared constant. In other words, the constant `1` should be defined in one place. In the translation string the constant would be used as `1 MB` but in the function it would be multiplied by 1000000.

Reply:
Oh, thanks. How does https://github.com/ampproject/amp-wp/pull/2509/commits/c966459de356d8d4280eedb550ce65bfae71fcbe look?

<img width="388" alt="new-notice-text" src="https://user-images.githubusercontent.com/4063887/59139598-aed80c80-8959-11e9-82dc-4fb4a36c31ab.png">